app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -31,26 +31,30 @@ def geocoder
 
       def report_errors(error)
         remapped_error = case error
-        when Faraday::Error,
-             ActionController::InvalidAuthenticityToken
-          :bad_request
-        else
-          :internal_server_error
-        end
+                         when Faraday::Error,
```

Comment:
personally, I like the less-indented version of this the case/when statements, I'm guessing our rubocop allows both

Reply:
You wrote the statements I'm changing here :) Yes, rubocop allows both, and I find the left-indented version confusing.

Reply:
if rubocop allows both, author's choice I think (until we can figure out how to get it to enforce one or the other)